apollo-router/src/plugins/traffic_shaping/deduplication.rs

```diff
@@ -99,6 +99,9 @@ where
 
                     match receiver.recv().await {
                         Ok(value) => {
+                            tracing::info!(
+                                monotonic_counter.apollo_router_deduplicated_queries_total = 1u64,
```

Comment:
Could you use the new macro instead ? Search for u64_counter in the codebase. And use the new conventions with dots. Example apollo.router...

Reply:
@bnjjj done!